pkg/icingadb/sla.go

```diff
@@ -0,0 +1,137 @@
+package icingadb
+
+import (
+	"context"
+	"github.com/icinga/icingadb/pkg/contracts"
+	v1 "github.com/icinga/icingadb/pkg/icingadb/v1"
+	"github.com/icinga/icingadb/pkg/types"
+	"github.com/pkg/errors"
+	"golang.org/x/sync/errgroup"
+	"time"
+)
+
+type SlaHistoryTrail struct {
+	Id                 types.Int `json:"id" db:"-"`
```

Comment:
Why is the column here at all?

Reply:
😕?

Reply:
Please see the review https://github.com/Icinga/icingadb/pull/566#pullrequestreview-1323482708 as a whole. The comments belong together.

Reply:
You're questioning why I have this column in this struct in the first place. I don't see any coherence with other comments.